src/llmcompressor/transformers/finetune/runner.py

```diff
@@ -237,15 +236,21 @@ def run_sequential_stages(self, checkpoint: Optional[str] = None):
                     output_dir=self._training_args.output_dir,
                     do_preprocess=do_preprocess,
                 )
+
+                if self.get_dataset_split("calibration") is not None:
```

Comment:
QQ: I'm assuming this handles not reprocessing the data twice?
What happens if the condition is None?

Reply:
Ill remove this if condition